frontend/src/components/QuestionTypes/QuestionList.vue

```diff
@@ -11,6 +11,11 @@ export default {
         return value.map((currentValue) => {
           if (currentValue.hasOwnProperty('name') && !currentValue.hasOwnProperty('text')) {
             currentValue.text = currentValue.name;
+          } else if (currentValue.hasOwnProperty('type') && currentValue.type === "separator") {
```

Comment:
In general, it would be nice to be able to draw a line across the entire selection (something similar to a b-`dropdown-divider` in the [`b-dropdown` component](https://bootstrap-vue.js.org/docs/components/dropdown/); however, that component cannot be bound...